File: LinkedList/Easy/268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        remaining=set()
        for i in range(len(nums)+1):
            remaining.add(i)
        print(remaining)
        for el in nums:
            remaining.discard(el)
        print(remaining)
        for el in remaining:
            return el


        

File: LinkedList/Easy/268-missing-number/missing-number.cpp
#include <unordered_set>

class Solution {
public:
    int missingNumber(vector<int>& nums) {
        std::unordered_set<int> found;
        for (int i=0; i<nums.size()+1; i++){
            found.insert(i);
        }

        for (int i=0; i<nums.size(); i++){
            found.erase(nums[i]);
        }
        for (int x : found) { 
            return x ;
            }
        return -1;
    }
};